packages/ra-ui-materialui/src/preferences/Configurable.stories.tsx

```diff
@@ -0,0 +1,203 @@
+);
+
+export const Basic = () => (
+    <StoreContextProvider value={memoryStore()}>
+        <PreferencesEditorContextProvider>
+            <Inspector />
+            <InspectorButton />
+            <hr />
+            <Box display="flex" alignItems="flex-start" gap="1em" padding="1em">
+                <ConfigurableTextBlock />
+                <ConfigurableSalesBlock />
+            </Box>
+        </PreferencesEditorContextProvider>
+    </StoreContextProvider>
+);
+
+export const Nested = () => (
+    <PreferencesEditorContextProvider>
+        <Inspector />
+        <InspectorButton />
+        <hr />
+        <Box display="flex" alignItems="flex-start" gap="1em" padding="1em">
+            <ConfigurableTextBlock>
+                <ConfigurableSalesBlock />
+            </ConfigurableTextBlock>
+        </Box>
+    </PreferencesEditorContextProvider>
+);
+
+export const MultipleInstances = () => (
+    <PreferencesEditorContextProvider>
+        <Inspector />
+        <InspectorButton />
+        <hr />
+        <Box display="flex" alignItems="flex-start" gap="1em" padding="1em">
+            <ConfigurableTextBlock preferenceKey="foo" />
+            <ConfigurableTextBlock preferenceKey="bar" />
+        </Box>
+    </PreferencesEditorContextProvider>
+);
+
+export const Unmount = () => {
+    const [isTextBlockVisible, setTextBlockVisible] = React.useState(true);
+    const [isSalesBlockVisible, setSalesBlockVisible] = React.useState(true);
+    return (
+        <PreferencesEditorContextProvider>
+            <Inspector />
+            <InspectorButton />
+            <hr />
+            <Box display="flex" alignItems="flex-start" gap="1em" padding="1em">
+                <Box flex="1" display="flex" flexDirection="column" gap="1em">
+                    <button
+                        onClick={() => setTextBlockVisible(!isTextBlockVisible)}
+                    >
+                        toggle text block
+                    </button>
+                    {isTextBlockVisible && <ConfigurableTextBlock />}
+                </Box>
+                <Box flex="1" display="flex" flexDirection="column" gap="1em">
+                    <button
+                        onClick={() =>
+                            setSalesBlockVisible(!isSalesBlockVisible)
+                        }
+                    >
+                        toggle sales block
+                    </button>
+                    {isSalesBlockVisible && <ConfigurableSalesBlock />}
+                </Box>
+            </Box>
+        </PreferencesEditorContextProvider>
+    );
+};
+export const I18n = () => {
+    const i18nProvider = {
+        translate: (key: string, options: any) => options?._ ?? key,
+        changeLocale: () => Promise.resolve(),
+        getLocale: () => 'en',
+    };
+    return (
+        <I18nContextProvider value={i18nProvider}>
+            <Basic />
+        </I18nContextProvider>
+    );
+};
+
+export const NotInContext = () => <TextBlock />;
```

Comment:
This story doesn't seem to do anything. Is not setting the background color from the Store